rest-api-spec/src/main/resources/rest-api-spec/test/indices.update_aliases/40_remove_with_must_exist.yml

```diff
@@ -0,0 +1,131 @@
+---
+"Throw aliases missing exception when removing non-existing alias with setting must_exist to true":
+  - skip:
+      version: " - 2.99.99"
+      reason: "introduced in 3.0"
+
+  - do:
+      indices.create:
+        index: test_index
+
+  - do:
+      indices.exists_alias:
+        name: test_alias
+
+  - is_false: ''
+
+  - do:
+      catch: /aliases \[test_alias\] missing/
+      indices.update_aliases:
+        body:
+          actions:
+            - remove:
+                index: test_index
+                alias: test_alias
+                must_exist: true
+
+  - do:
+      catch: /aliases \[testAlias\*\] missing/
+      indices.update_aliases:
+        body:
+          actions:
+            - remove:
+                index: test_index
+                aliases: [ testAlias* ]
+                must_exist: true
+
```

Comment:
@gaobinlong could you please add one test case for empty aliases (here or in `10_basic`), thank you:
```
  - do:
      catch: /\[aliases\] can't be empty/
      indices.update_aliases:
        body:
          actions:
            - remove:
                index: test_index
                aliases: []
                must_exist: true
```

Reply:
I've added that, thanks!